src/SFA.DAS.EmployerUsers.Support.Web/Global.asax.cs

```diff
@@ -10,12 +10,13 @@
 using SFA.DAS.Support.Shared.SiteConnection;
 using SFA.DAS.Web.Policy;
 
+//using AuthorizeAttribute = System.Web.Http.AuthorizeAttribute;
```

Comment:
Does this need to be removed?

Reply:
yes, removing it now. The whole problem was because this is a mixed MVC/WebAPI pipeline Application and each controller needs its own attribute type. A Globally applied attributes screws it all up.